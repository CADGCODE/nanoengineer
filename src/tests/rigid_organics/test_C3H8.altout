======= test_C3H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
error reading atom 0 from test_C3H8.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C3H8.mmp  > test_C3H8.xyz
final RMS gradient=2292.736375 after 6 iterations
== structure comparison ==
======= test_C3H8.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C3H8.mmp
# Output File: test_C3H8.xyz
# Trace File: test_C3H8.trc
# Number of Atoms: 11
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 3861.76 5280.51
1 2415.12 2774.42
2 2295.31 2652.62
3 2292.77 2650.02
4 2292.74 2649.99
5 2292.74 2649.99
6 2292.74 2649.99
# Done: Minimization final rms: 2292.736375, highForce: 2649.986184
======= test_C3H8.xyz =======
11
RMS=2292.736375
C 0.447116 -1.285855 0.011972
C -0.460821 -0.002140 0.015227
C 0.413139 1.305287 0.019604
H -0.195096 -2.189409 0.027923
H 1.105650 -1.291031 0.896944
H 1.071825 -1.308719 -0.899125
H -1.098604 -0.014051 0.901392
H -1.100586 -0.006231 -0.873647
H -0.255207 2.188949 0.062263
H 1.021189 1.363188 -0.900933
H 1.084395 1.314013 0.896379
