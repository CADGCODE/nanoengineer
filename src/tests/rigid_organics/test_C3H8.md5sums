stdout 554EA46702AC15C3C4165CBF4A3EAB98
stderr D41D8CD98F00B204E9800998ECF8427E
test_C3H8.xyz 60447FAB323D73BFFE274AA8F3023391
