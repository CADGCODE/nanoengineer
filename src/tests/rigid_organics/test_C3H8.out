======= test_C3H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
